### ISSUE TITLE
L10n tests fail with: Cannot find 'AccessibilityIdentifiers' in scope

### ISSUE THREAD

**Issue by isabelrios** [mozilla-mobile/firefox-ios#9457]
After the [PR adding an Accessibility Identifiers struct](https://github.com/mozilla-mobile/firefox-ios/pull/9411/) landed, L10n tests started to fail. The AccessibilityIdentifier.swift file was not added to the L10n target and it is needed so that it compiles.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-3398)
